### PR TITLE
Update sample app with new features

### PR DESCRIPTION
**Pull request by nicklas-dohrn**
This changes and adds two main features:
- The last sent log message is now shown in the Client UI
- The currently active service bindings are listed in the UI

### CODE REVIEW

File: sample/cf-nodejs-logging-support-sample/app.js [hunk 1]
```diff
@@ -89,9 +102,23 @@ app.get("/correlationandtenantid", function (req, res) {
...
   var tenantId = reqLogger.getTenantId();
   reqLogger.info("Correlation ID: %s Tenant ID: %s", correlationId, tenantId);
-  res.send();
+  res.send(lastMessage);
 });
 
+app.get("/binding_information", function (_, res) {
+  result = "There are the following bindings: <br>"
+  if (process.env.VCAP_SERVICES)
+    var services = JSON.parse(process.env.VCAP_SERVICES)
```
Comment (christiand93) [inline sample/cf-nodejs-logging-support-sample/app.js:111]: I would not define a variable inside a block and then consume it outside. Technically, this is working for a `var` variable as it is function scoped but it looks a bit strange.

File: sample/cf-nodejs-logging-support-sample/app.js [hunk 2]
```diff
@@ -89,9 +102,23 @@ app.get("/correlationandtenantid", function (req, res) {
...
+  console.log(services)
+  if (services == undefined)
+    res.send("There are no bindings present for this application")
+  else {
+    for (var service in services)
+      result += service + "<br>"
+    res.send(result)
```
Comment (christiand93) [inline sample/cf-nodejs-logging-support-sample/app.js:118]: Just to confirm: This prints the names of the services only.

File: sample/cf-nodejs-logging-support-sample/public/index.html [hunk 3]
```diff
@@ -27,9 +27,18 @@ <h2 class="subtitle">Buttons to test functionalities:</h2>
...
+            <div class="subtitle">The message sent:</div>
+            <div class="feedback-box" id="feedback">
+            </div>
+            <div class="subtitle">Binding information:</div>
+            <div class="feedback-box" id="binding_information">
             <a href="https://sap.github.io/cf-nodejs-logging-support/">Visit Documentation</a>
         </div>
     </div>
+
+
```
Comment (christiand93) [inline sample/cf-nodejs-logging-support-sample/public/index.html:41]:
```suggestion
```

**Review by christiand93 — CHANGES_REQUESTED** [review]

File: sample/cf-nodejs-logging-support-sample/app.js (hunk 2 above)
Comment (nicklas-dohrn) [reply sample/cf-nodejs-logging-support-sample/app.js:118]: Yes, this only gets the service names

File: sample/cf-nodejs-logging-support-sample/app.js (hunk 1 above)
Comment (nicklas-dohrn) [reply sample/cf-nodejs-logging-support-sample/app.js:111]: definition is factored out

File: sample/cf-nodejs-logging-support-sample/manifest.yml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 applications:
-- name: cf-nodejs-logging-support-sample
+- name: nodejs_sample_app_nicklas
```
Comment (christiand93) [inline sample/cf-nodejs-logging-support-sample/manifest.yml:3]:
```suggestion
- name: cf-nodejs-logging-support-sample
```

**Review by christiand93 — APPROVED** [review]
Nice, see comment below

**Pull request merged**